_includes/stories.html

```diff
@@ -0,0 +1,26 @@
+<div class="stories">
+  {% assign stories = site.stories | where:"lang", page.lang | where:"show", true | sort: "index" %}
+  {% for story in stories %}
+  <article class="stories__story" id="story-{{ story.index }}">
+    <div class="stories__story-column-left">
+      <img
+        src="{{ relroot }}img/story-images/{{ story.graphic }}"
+        alt="{{story.alt}}"
+        class="stories__story-image"
+      >
+    </div>
+    <div class="stories__story-column-right">
+      <h2 class="stories__story-title-small">
```

Comment:
Sommige headings komen twee keer voor (“Hoe werkt het?” en “Voorbeeld”), en de headingteksten beschrijven niet de inhoud die erna komt (zie [No content between headings](https://hiddedevries.nl/en/blog/2020-09-05-when-there-is-no-content-between-headings)).

In plaats van “Voorbeeld”, “Voorbeeld” zou bv “Je zat naast iemand”, “Je fietste iemand voorbij” beter zijn, uniek en schept een verwachting van wat er in deze sectie precies zal worden uitgelegd.

Reply:
Opgelost met de volgende constructie: "Voorbeeld: Je zat naast iemand"

```html
<h2>
        <span class="stories__story-title__highlight">{{story.title-highlight}}</span>
        <span class="stories__story-title">{{story.title}}</span>
</h2>
```